spec/rhc/commands/ssh_spec.rb

```diff
@@ -171,6 +171,26 @@
     end
   end
 
+  describe 'app ssh custom ssh with inaccessible path' do
+    let(:arguments) { ['app', 'ssh', 'app1', '--ssh', '/etc/'] }
+
+    before(:each) do
```

Comment:
In the other tests, we have `context` blocks with "when run" or something more specific.  Might as well do the same here for consistency.